lib/graphqlHelpers.js

```diff
@@ -49,7 +48,7 @@ const getMultisig = async (address, chainId) => {
     data: {
       query: `
         query {
-          getMultisig(address: "${address}", chainId: "${chainId}",) {
+          getMultisig(address: "${address}") {
```

Comment:
Although unlikely, it is possible for an address to be the same across chains, so I'd rather be prepared than have it break if someone attempts to use it in that way.

Reply:
Hi, thanks for the quick feedback! It was impossible for me to query chainid with the provided graphql scheme, that's why I changed it.

Reply:
What issue were you running into?

Reply:
I just couldn't query the chainid, even tried on faunadbs playground. That's why I got errors

Reply:
Probably you have a different scheme uploaded?

Reply:
this is the schema thats used https://github.com/cosmos/cosmos-multisig-ui/blob/master/db-schema.graphql

Good catch! i need to update the query in that schema file to include the chainId

Reply:
@samepant feel free to fix this in your PR branch and we close here. Then we can continue testing tomorrow.

Reply:
@msteiner96 is a buddy from Confio by the way